Assets/Scripts/Interaction/Hand.cs

```diff
@@ -164,4 +177,35 @@ public void switchPrimaryHand()
         IS_PRIMARY = !IS_PRIMARY;
         initialize();
     }
+
+    public void counterProjectile() {
+        timeSlowed = 1f;
+        Time.timeScale = 0.333333f;
+        Time.fixedDeltaTime = 0.02f * Time.timeScale;
+        if (slowUI == null) {
+            slowUI = Instantiate<GameObject>(Resources.Load<GameObject>("Prefabs/CounterProjectile"));
+            slowUI.transform.position = GameObject.FindGameObjectWithTag("Player").transform.forward*1.1f;
+        }
+    }
+
+    void updateCounterSlowTime() {
```

Comment:
you could use a coroutine rather than calling this in update every frame. Another, possibly better option is the Invoke command, which calls a functino after n seconds.